src/qibosoq/qick_programs.py

```diff
@@ -559,9 +542,9 @@ def initialize(self):
 
 
 SWEEPERS_TYPE = {
-    Parameter.frequency: "freq",
-    Parameter.amplitude: "gain",
-    Parameter.bias: "gain",
-    Parameter.relative_phase: "phase",
-    Parameter.start: "t",
+    Parameter.FREQUENCY: "freq",
+    Parameter.AMPLITUDE: "gain",
+    Parameter.BIAS: "GAin",
```

Comment:
Maybe the case is incorrect, isn't it?
```suggestion
    Parameter.BIAS: "Gain",
```